src/host/tcp.h

```diff
@@ -162,6 +162,12 @@ namespace asynchost
         case CONNECTED:
           return send_write(req, len);
 
+        case DISCONNECTED:
```

Comment:
The changes in this PR unveiled one issue: when a RPC client (here the joining node) connects to a CCF node and the client is killed half-way through a request being processed by the server node, the `tls_outbound` messages emitted by the enclave (on the server side) could kill the server node (because of the `default:` of this switch on line 171).